easybuild/framework/easyblock.py

```diff
@@ -227,9 +227,15 @@ def __init__(self, ec):
 
         # try and use the specified group (if any)
         group_name = build_option('group')
-        if self.cfg['group'] is not None:
-            self.log.warning("Group spec '%s' is overriding config group '%s'." % (self.cfg['group'], group_name))
-            group_name = self.cfg['group']
+        group_spec = self.cfg['group']
+        if group_spec is not None:
+            if isinstance(group_spec, tuple):
+                if len(group_spec) == 2:
+                    group_spec = group_spec[0]
+                else:
+                    raise EasyBuildError("Found group spec in tuple format that is not a 2-tuple: %s", ec_group)
```

Comment:
this is not correct:

* uses non-existing variable `ec_group`
* string templating with a tuple value (`group_spec`) doesn't work for a single `%s`, should be `str(group_spec)`

fixed in https://github.com/damianam/easybuild-framework/pull/11, which includes an enhanced test